src/scores/dates.py

```diff
@@ -2,15 +2,15 @@
 from pandas import DataFrame, Timestamp
 
 
-def main(df: DataFrame) -> DataFrame:
+def main(checks: DataFrame) -> DataFrame:
     """Draft function for scoring date values within dataset.
```

Comment:
Docstring might be out of date - is this still draft? Might be worth adding a TODO statement if it is so it's easier to find and fix in the future?

Reply:
I put this in as a placeholder score from the beginning, I just updated `dates` and `languages` with all the scores that are possible to do at this point.